keyboards/planck/planck.c

```diff
@@ -12,8 +12,8 @@ const keypos_t hand_swap_config[MATRIX_ROWS][MATRIX_COLS] = {
 
 __attribute__ ((weak))
 void matrix_init_kb(void) {
-	// Turn status LED on
-	#ifdef __AVR__
+	// Turn status LED on, with the exception of THK
+	#if defined(__AVR__) && !defined(__AVR_ATmega32A__)
```

Comment:
Might as well just make it for the atmega32u4.  The planck light doesn't use E6 for the LED (it uses D6), and the rest of the planck revisions dont' use it either.   rev1-5 all use atmega32u4's. 
```suggestion
	#if defined(__AVR_ATmega32U4__)
```